Resources/Textures/Objects/Weapons/Melee/combat_knife.rsi/meta.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken/modified from tgstation at commit https://github.com/tgstation/tgstation/pull/30680/commits/a494d500b42bc8c7275c57219e08e65b825e94f6",
```

Comment:
is this correct? i thought you sprited this yourself?

Reply:
Yeah, I used the tg combat knife as an inspiration.  Also, already made the sprite smaller by reducing the handle and having it centered.